content/documentation/server/gnatsd-authorization.md

```diff
@@ -1,96 +1,71 @@
-publish             // Singleton or array
-subscribe           // Singleton or array
-```
-
-## Example
-
-For example, since Alice is an ADMIN she can publish/subscribe on any subject. We use the wildcard “>” to match any subject.
+You set permissions by creating an entry inside of the `authorization` configuration block that conforms to the following syntax:
 
 ```
 authorization {
-  ADMIN = {
-    publish = ">"
-    subscribe = ">"
-  }
-}
-```
-
-Bob is REQUESTOR and can publish requests on subjects req.foo or req.bar, and subscribe to anything that is a response (_INBOX.*).
-
-```
-authorization {
-  REQUESTOR = {
-    publish = ["req.foo", "req.bar"]
-    subscribe = "_INBOX.*"
+  PERMISSION_NAME = {
+    publish = "singleton" or ["array", ...]
+    subscribe = "singleton" or ["array", ...]
   }
 }
 ```
 
-Note that the publish field is an array with two subjects; the subscribe field is a singleton. The parser is generous at understanding what the intent is, so arrays and singletons are processed.
+# Example
 
-Joe has no permission grant and therefore inherits the default permission set. You set the inherited default permissions by assigning them to the default_permissions entry inside of the authorization configuration block.
+Here is an example authorization configuration that defines four users, three of whom are assigned explicit permissions.
 
 ```
 authorization {
-  default_permissions = {
-    publish = "SANDBOX.*"
-    subscribe = ["PUBLIC.>", "_INBOX.>"]
-  }
-}
-```
-
-Note that `_INBOX.*` subscribe permissions must be granted in order to use the request APIs in the Synadia supported clients. If an unauthorized client publishes or attempts to subscribe to a subject, the action fails and is logged at the server, and an error message is returned to the client.
-
-## Complete example
-
-```
-authorization {
-  
   ADMIN = {
     publish = ">"
     subscribe = ">"
   }
-  
   REQUESTOR = {
     publish = ["req.foo", "req.bar"]
-    subscribe = "_INBOX.*"
+    subscribe = "_INBOX.>"
+  }
+  RESPONDER = {
+    subscribe = ["req.foo", "req.bar"]
+    publish = "_INBOX.>"
   }
-  
   DEFAULT_PERMISSIONS = {
     publish = "SANDBOX.*"
     subscribe = ["PUBLIC.>", "_INBOX.>"]
   }
-  
+
   PASS: abcdefghijklmnopqrstuvwxwz0123456789
-  
   users = [
-    {user: alice, password: foo, permissions: $ADMIN}
-    {user: bob,   password: bar, permissions: $REQUESTOR}
-    {user: joe,   password: $PASS}
+    {user: joe,     password: foo,   permissions: $ADMIN}
+    {user: alice,   password: bar,   permissions: $REQUESTOR}
+    {user: bob,     password: $PASS, permissions: $RESPONDER}
+    {user: charlie, password: bar}
   ]
 }
 ```
 
-Note that the variable identifier (name) is not case sensitive, but is capitalized by convention for readability. The variable is referenced using the $ character.
\ No newline at end of file
+Since Joe is an ADMIN he can publish/subscribe on any subject. We use the wildcard “>” to match any subject.
```

Comment:
"Since Joe has been assigned ADMIN permissions..." - nitpick

Reply:
leaving this as it matches the config file on gnatsd README.md